proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -1481,11 +1481,14 @@
           "fill-1": { "$value": "{purmerend.color.blue-41}" },
           "fill-2": { "$value": "{purmerend.color.blue-41}" },
           "text-1": { "$value": "{purmerend.color.blue-44}" },
-          "text-2": { "$value": "{purmerend.color.blue-44}" }
+          "text-2": { "$value": "{purmerend.color.blue-44}" },
+          "text-3": { "$value": "{purmerend.color.blue-42}" }
         },
         "primary-inverse": {
           "bg-1": { "$value": "{purmerend.color.cyan-6}" },
           "bg-2": { "$value": "{purmerend.color.cyan-6}" },
+          "bg-alert": {"$value": "{purmerend.color.blue-41}"},
+          "bg-textarea": {"$value": "{purmerend.color.blue-41}"},
```

Comment:
Waarom zit de textarea in een algemene scale? Moet die niet gewoon de "text" scale uitlezen?